smack-integration-test/src/main/java/org/igniterealtime/smack/inttest/XmppConnectionManager.java

```diff
@@ -184,20 +184,18 @@ public static boolean removeConnectionDescriptor(String nickname) {
 
         switch (sinttestConfiguration.accountRegistration) {
         case serviceAdministration:
-        case inBandRegistration:
             accountRegistrationConnection = defaultConnectionDescriptor.construct(sinttestConfiguration);
             accountRegistrationConnection.connect();
             accountRegistrationConnection.login(sinttestConfiguration.adminAccountUsername,
```

Comment:
Couldn't we simply move this line into the `else` block below?

Reply:
Maybe? Having an if/else block in a switch/case that uses the same arguments smells. The fact that I have to think if it would be a suitable alternative makes me prefer my original solution: The code is a lot easier to understand, and isn't even any longer. That goes a long way in preventing the kind of bugs that this PR is fixing.

Reply:
This approach causes 8 insertions and 9 deletions (if you don't count the deletion of the empty line 194). The reduction in deletions comes due the removal of the if/else block, which accounts for 3 deleted lines.

However, it introduces some duplicate code. And the change itself is much more verbose as compared to https://github.com/igniterealtime/Smack/pull/573.

I prefer to avoid duplicate code and hence tend to to with https://github.com/igniterealtime/Smack/pull/573 over this.

But in any case, thanks for the bug report!